### ISSUE TITLE
lint: parenthesis checker panics on cmd/internal/obj/x86

### ISSUE THREAD

**Issue by Quasilyte** [PieselBois/kfulint#53]
```
./bin/kfulint -dir=$GOROOT/src/cmd/internal/obj/x86
panic: ast.Walk: unexpected node type <nil> [recovered]
	panic: ast.Walk: unexpected node type <nil>

goroutine 51 [running]:
main.(*linter).CheckFile.func1.1(0xc00056ea90, 0x69a792, 0xb, 0x6c9300, 0xc000504160)
	src/github.com/PieselBois/kfulint/cmd/kfulint/main.go:139 +0x14e
panic(0x63cd20, 0xc000158480)
	$GOROOTsrc/runtime/panic.go:494 +0x1b9
go/ast.Walk(0x6c97c0, 0xc000158430, 0x0, 0x0)
	$GOROOTsrc/go/ast/walk.go:364 +0x311f
go/ast.Inspect(0x0, 0x0, 0xc000158430)
	$GOROOTsrc/go/ast/walk.go:385 +0x4b
github.com/PieselBois/kfulint/lint.(*ParenthesisChecker).validateType(0xc000504160, 0x0, 0x0)
	src/github.com/PieselBois/kfulint/lint/parenthesis_checker.go:63 +0x6b
github.com/PieselBois/kfulint/lint.(*ParenthesisChecker).Check(0xc000504160, 0xc0000ac280, 0xc00056ea90, 0x69a792, 0xb)
	src/github.com/PieselBois/kfulint/lint/parenthesis_checker.go:47 +0x1f2
main.(*linter).CheckFile.func1(0xc00056ea90, 0xc0000ac280, 0xc000022340, 0x69a792, 0xb, 0x6c9300, 0xc000504160)
	src/github.com/PieselBois/kfulint/cmd/kfulint/main.go:142 +0xd5
created by main.(*linter).CheckFile
	src/github.com/PieselBois/kfulint/cmd/kfulint/main.go:126 +0xc9
```

**Comment by fexolm** [comment]
I will try to fix it today.

**Comment by Quasilyte** [comment]
Fixed in #58.
